src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -140,7 +140,13 @@ class JesBackend extends Backend with LazyLogging {
    * Delete authentication file in gcs once workflow is in a terminal state.
    */
   override def cleanUpForWorkflow(workflow: WorkflowDescriptor)(implicit ec: ExecutionContext): Future[Unit] = {
-    Future(JesConnection.storage.deleteObject(GoogleCloudStoragePath(gcsAuthFilePath(workflow))))
+    // TODO have this retry and/or check for the file's existence explicitly
+    Try(JesConnection.storage.deleteObject(GoogleCloudStoragePath(gcsAuthFilePath(workflow)))) match {
+      case Failure(f) => logger.warn(s"Could not delete GCS file ${gcsAuthFilePath(workflow)}: ${f.getMessage}")
+      case Success(_) =>
+    }
```

Comment:
Gotcha... I'll factor that in.  I still want to figure out how to query GCS for an objects existence first and add a retry here which might obviate the need for recover.  But if not, I'll add the `recover`